docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -163,5 +163,5 @@ Native date (`type="date"`), time (`type="time"`) and date&time (`type="datetime
 Some test environments (i.e. `jsdom`) do not support media query. In such cases, components will be rendered in desktop mode. To modify this behavior you can fake the `window.matchMedia`.
 :::
 
-Be aware that running tests in headless browsers might not pass our default mediaQuery (`pointer: fine`).  
+Be aware that running tests in headless browsers might not pass our default mediaQuery (`pointer: fine`).
```

Comment:
```suggestion
Be aware that running tests in headless browsers might not pass the default mediaQuery (`pointer: fine`).

```
Double space at the end of a line produces a new line/paragraph—to maintain existing layout we should introduce a new empty line.

Reply:
Fair point. New line is not necessary here. 👌 
I've just added it purely because of styling—where only `In` word is visible in the same line of the previous sentence.